SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/SpatialTestInitialOnly/SpatialTestInitialOnlyForInterestActor.cpp

```diff
@@ -35,21 +35,16 @@ void ASpatialTestInitialOnlyForInterestActor::PrepareTest()
 	AddStep(TEXT("Init test environment"), FWorkerDefinition::Server(1), nullptr, [this]() {
 		// Spawn cube
 		ASpatialTestInitialOnlySpawnActor* SpawnActor = GetWorld()->SpawnActor<ASpatialTestInitialOnlySpawnActor>(
-			FVector(-400.0f, 0.0f, 40.0f), FRotator::ZeroRotator, FActorSpawnParameters());
+			FVector(-1500.0f, 0.0f, 40.0f), FRotator::ZeroRotator, FActorSpawnParameters());
 
 		RegisterAutoDestroyActor(SpawnActor);
 
-		// Set the PositionUpdateThresholdMaxCentimeters to a lower value so that the spatial position updates can be sent every time the
-		// character moves, decreasing the overall duration of the test
-		PreviousMaximumDistanceThreshold = GetDefault<USpatialGDKSettings>()->PositionUpdateThresholdMaxCentimeters;
-		GetMutableDefault<USpatialGDKSettings>()->PositionUpdateThresholdMaxCentimeters = 0.0f;
-
```

Comment:
This is not necessary, jumps as large as these should always force a spatial position update.

Reply:
hmmmmmmmmmmmmmmmmmmmmmmm, actually I am wrong
the default max treshold is 5000 units.... seems kinda large.
I think the proper fix here would be to add this to the base settings used for every map.
Also, interestingly, the tests complete quickly even without this, so something must be updating the position?